composer.json

```diff
@@ -38,21 +37,16 @@
     },
     "suggest": {
         "ext-curl": "If you use CURLRequest class",
-        "ext-dom": "If you use TestResponse",
         "ext-exif": "If you run Image class tests",
-        "ext-fileinfo": "Improves mime type detection for files",
         "ext-gd": "If you use Image class GDHandler",
         "ext-imagick": "If you use Image class ImageMagickHandler",
-        "ext-libxml": "If you use TestResponse",
         "ext-memcache": "If you use Cache class MemcachedHandler with Memcache",
         "ext-memcached": "If you use Cache class MemcachedHandler with Memcached",
         "ext-mysqli": "If you use MySQL",
         "ext-oci8": "If you use Oracle Database",
         "ext-pgsql": "If you use PostgreSQL",
         "ext-readline": "Improves CLI::input() usability",
         "ext-redis": "If you use Cache class RedisHandler",
-        "ext-simplexml": "If you format XML",
-        "ext-sqlite3": "If you use SQLite3",
```

Comment:
Can be disabled.
https://www.php.net/manual/en/sqlite3.installation.php

Reply:
Alright, so this goes for "Can be disabled" remarks.

The following:

> This extension is enabled by default. It may be disabled by using the following option at compile time: --disable-simplexml

, translates to: official versions of PHP include simplexml, but when building PHP from source there is the option to disable simplexml.
Basically simplexml it's always there with the official versions of PHP.

Framework decision required.

Reply:
I forgot to add the critical part.
The extensions which **can be disabled** are not supplied by the package managers (i.e. dnf).

In case someone builds PHP from source using i.e. --disable-simplexml and later on decides he needs ext-simplexml, then he has to build from source simplexml. Now, this is odd indeed.

Reply:
These are not linux distribution's package names, but PHP extensions.

```console
$ composer show --platform | grep ^ext
ext-bcmath                 8.1.26   The bcmath PHP extension
ext-bz2                    8.1.26   The bz2 PHP extension
ext-calendar               8.1.26   The calendar PHP extension
ext-ctype                  8.1.26   The ctype PHP extension
ext-curl                   8.1.26   The curl PHP extension
ext-date                   8.1.26   The date PHP extension
ext-dba                    8.1.26   The dba PHP extension
ext-dom                    20031129 The dom PHP extension
ext-exif                   8.1.26   The exif PHP extension
ext-ffi                    8.1.26   The FFI PHP extension
ext-fileinfo               8.1.26   The fileinfo PHP extension
ext-filter                 8.1.26   The filter PHP extension
ext-ftp                    8.1.26   The ftp PHP extension
ext-gd                     8.1.26   The gd PHP extension
ext-gettext                8.1.26   The gettext PHP extension
ext-gmp                    8.1.26   The gmp PHP extension
ext-hash                   8.1.26   The hash PHP extension
ext-iconv                  8.1.26   The iconv PHP extension
ext-imagick                3.7.0    The imagick PHP extension
ext-intl                   8.1.26   The intl PHP extension
ext-json                   8.1.26   The json PHP extension
ext-ldap                   8.1.26   The ldap PHP extension
ext-libxml                 8.1.26   The libxml PHP extension
ext-mbstring               8.1.26   The mbstring PHP extension
ext-memcached              3.2.0    The memcached PHP extension
ext-mysqli                 8.1.26   The mysqli PHP extension
ext-mysqlnd                0        The mysqlnd PHP extension (actual version: mysqlnd 8.1.26)
ext-odbc                   8.1.26   The odbc PHP extension
ext-openssl                8.1.26   The openssl PHP extension
ext-pcntl                  8.1.26   The pcntl PHP extension
ext-pcre                   8.1.26   The pcre PHP extension
ext-pdo                    8.1.26   The PDO PHP extension
ext-pdo_dblib              8.1.26   The pdo_dblib PHP extension
ext-pdo_mysql              8.1.26   The pdo_mysql PHP extension
ext-pdo_odbc               8.1.26   The PDO_ODBC PHP extension
ext-pdo_pgsql              8.1.26   The pdo_pgsql PHP extension
ext-pdo_sqlite             8.1.26   The pdo_sqlite PHP extension
ext-pgsql                  8.1.26   The pgsql PHP extension
ext-phar                   8.1.26   The Phar PHP extension
ext-posix                  8.1.26   The posix PHP extension
ext-pspell                 8.1.26   The pspell PHP extension
ext-readline               8.1.26   The readline PHP extension
ext-redis                  5.3.7    The redis PHP extension
ext-reflection             8.1.26   The Reflection PHP extension
ext-session                8.1.26   The session PHP extension
ext-shmop                  8.1.26   The shmop PHP extension
ext-simplexml              8.1.26   The SimpleXML PHP extension
ext-soap                   8.1.26   The soap PHP extension
ext-sockets                8.1.26   The sockets PHP extension
ext-sodium                 8.1.26   The sodium PHP extension
ext-spl                    8.1.26   The SPL PHP extension
ext-sqlite3                8.1.26   The sqlite3 PHP extension
ext-sqlsrv                 5.11.0   The sqlsrv PHP extension
ext-sysvmsg                8.1.26   The sysvmsg PHP extension
ext-sysvsem                8.1.26   The sysvsem PHP extension
ext-sysvshm                8.1.26   The sysvshm PHP extension
ext-tidy                   8.1.26   The tidy PHP extension
ext-tokenizer              8.1.26   The tokenizer PHP extension
ext-xdebug                 3.2.1    The xdebug PHP extension
ext-xml                    8.1.26   The xml PHP extension
ext-xmlreader              8.1.26   The xmlreader PHP extension
ext-xmlwriter              8.1.26   The xmlwriter PHP extension
ext-xsl                    8.1.26   The xsl PHP extension
ext-zend-opcache           8.1.26   The Zend OPcache PHP extension
ext-zip                    1.19.5   The zip PHP extension
ext-zlib                   8.1.26   The zlib PHP extension
```

Reply:
Oh, there is `ext-json`.

Reply:
I think this PR is not needed.